src/components/grapedown/renderers.js

```diff
@@ -28,6 +28,15 @@ export function renderTag(tag, props, children) {
     if (isGrapeUrl(href)) {
       return createElement(GrapeObject, props, children)
     }
+
+    // Split chat pm path and user id
+    const [, chatPmPath, pmUserId] = href.match(/^(\/chat\/pm)\/(\d+)?\/$/) || []
```

Comment:
Can we do this somehow without `match`? For example path to this function `match` prop from react-router?

Reply:
`match` was the most pragmatic way I could come up with to split chat pm path and user id out of the string. (see https://github.com/ubergrape/grape-web-client/pull/683/commits/ecc8315ad135af0acdc4e61203c9e4e3ea1e8e75 & https://github.com/ubergrape/grape-web-client/pull/683/commits/b5a5ceecfdd96ea69437e9d93883608369821425).
Not sure I understand how `react-router` would make this simpler. Could you provide an example? Thanks! 🙌

Reply:
@AleshaOleg Steps to reproduce:
- Quote own message -> Your name won't be wrapped in a link in the message footer
- Quote somebody else's message -> Name is wrapped in a link to PM in the message footer

Couldn't find in the link how this one liner would be improved by the router. Could you paste here a concrete example?
Also with the selector, can I just pass the href and it outputs me both `chatPmPath` and `pmUserId`?
If not, then my gut feeling is that it could be much harder to read and a bit over-engineered just by splitting `/chat/pm` and `userId` out of a string.

Reply:
@AleshaOleg Oh! Yeah right `href` is a bit confusing here. It was a URL at first, but then had to make it a path to make the router work properly. `?\/` here in the end of regex supposed to be an optional `/` in the end of the path. So it can parse both from `/chat/pm/13721` and `/chat/pm/13721/`.

Reply:
but for me `/chat/pm/13721` now gives two `undefined` values and for `/chat/pm/13721/` - now is ok.

Reply:
Oh, ofc! It should be `\/?`. Will fix asap.